modules/nf-core/survivor/merge/main.nf

```diff
@@ -0,0 +1,75 @@
+process SURVIVOR_MERGE {
+    tag "$meta.id"
+    label 'process_low'
+
+    conda "bioconda::survivor=1.0.7"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/survivor:1.0.7--h9a82719_1':
+        'quay.io/biocontainers/survivor:1.0.7--h9a82719_1' }"
+
+    input:
+    tuple val(meta), path(vcfs)
+
+    output:
+    tuple val(meta), path("*.vcf")   , emit: vcf
+    path "versions.yml"              , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: [:]
+    def prefix = task.ext.prefix ?: "${meta.id}"
+
+    def max_distance_breakpoints        = args.max_distance_breakpoints         ?: 0.2
+    def min_supporting_callers          = args.min_supporting_callers           ?: 1
+    def account_for_type                = args.account_for_type                 ?: 0
+    def account_for_sv_strands          = args.account_for_sv_strands           ?: 0
+    def estimate_distanced_by_sv_size   = args.estimate_distanced_by_sv_size    ?: 0
+    def min_sv_size                     = args.min_sv_size                      ?: 20
```

Comment:
Given these are for positional arguments, I would say to make the module much less brittle I would use `input` args for these.

You should not set hidden defaults in pipelines either, which is why I tihnk this is a bad idea.

Reply:
Okay fair point there. These arguments all need to be used in a flexible way so this will be a long `input` channel :grin: